tests/Runtime-MultiDex/Resources/Resource.designer.cs

```diff
@@ -1,270 +0,0 @@
-#pragma warning disable 1591
```

Comment:
Why is this file removed, when `Mono.Android-TestsMultiDex.csproj` was updated to add this file to `@(Compile)`?